.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,86 +1,48 @@
-name: Bug Report
-description: Report a problem with TinyUSB
-labels: 'Bug 🐞'
+name: Bug report
+description: Report build and runtime bugs
+labels: ["Type: Bug"]
 body:
-  - type: markdown
-    attributes:
-      value: |
-        Thanks for taking the time to fill out this bug report!
-        It's okay to leave some blank if it doesn't apply to your problem.
-
-  - type: dropdown
+  - type: checkboxes
+    id: checklist
     attributes:
-      label: Operating System
+      label: Answers checklist.
+      description: Before submitting a new issue, please follow the checklist and try to find the answer.
       options:
-              - Linux
-              - MacOS
-              - RaspberryPi OS
-              - Windows 7
-              - Windows 10
-              - Windows 11
-              - Others
-    validations:
-      required: true
-
+        - label: I have read the component documentation [ESP-IDF Components](https://components.espressif.com) and the issue is not addressed there.
+          required: true
+        - label: I am using target and esp-idf version as defined in component's idf_component.yml
+          required: true
+        - label: I have searched the [issue tracker](https://github.com/espressif/tinyusb/issues?q=is%3Aissue) for a similar issue and not found any related issue.
```

Comment:
I suppose, the `Issues` tab will be created in this repo after merging this changes, as right now the tab is missing and this link gets you to `Pull requests` tab.

Reply:
No, Issues tab should be created by maintainers. 

So, the questions is: Do we want split the issues for espressif/tinyusb? 

Basically, right now, there are three options: 
1. Use `esp-usb` issue tracker
2. Use `esp-idf` issue tracker
3. Create and use `espressif/tinyusb` issue tracker

I guess that this questions is being asked all the time, when new commiter comes.
Anyway, @tore-espressif do we have any special requirements why we didn't create the issue tracker for `espressif/tinyusb`?
Also, please, share your opinion about question which tracker to use. Thanks.

Reply:
Oh sry, I haven't thought about the issues tracker in the tinyusb upstream. 

I took a look at the other espressif github repos, which are forked, just like espressif/tinyusb, [here](https://github.com/espressif?q=&type=fork&language=&sort=). 

Some espressif forks have the issue trackers, some don't. Sometimes, even the upstream repo does not have the issue tracker, so the espressif fork has to have one. Looks like, it's rather a personal choice to have one.